### ISSUE TITLE
maximiizedMargin issue when set to 0

### ISSUE THREAD

**Issue by whittssg** [Flyer53/jsPanel3#42]
Hello,

When i set this:

```
 maximizedMargin: {
        top:    0,
        right:  0,
        bottom: 0,
        left:  0 ,
    },
```

I still get 5px margins on all sides. I have to set it to 0.1 or something similar to get it close to 0.

Thanks,

**Comment by Flyer53** [comment]
```javascript
$.jsPanel({
    maximizedMargin: 0
});
```
should do it. Also
```javascript
$.jsPanel({
    maximizedMargin: [0,0,0,0]
});
```
works fine.
I need to check the code why using an object doesn't work as intended. But simply **maximizedMargin: 0** is the shortest way anyhow.

**Comment by Flyer53** [comment]
**Tip:**
If you want this to be a default for **all** your panels call
```javascript
$.jsPanel.defaults.maximizedMargin = 0;
```
after loading the jsPanel script and before you create the first panel to change the global jsPanel default value for option.maximizedMargin.
Then you don't have to worry about it anymore in the individual panel configs.

**Comment by whittssg** [comment]
Thanks. maximizedMargin: 0 worked fine.

**Comment by Flyer53** [comment]
Fixed with v3.8.0 😄
